src/lwc/geBatchGiftEntryHeader/__tests__/geBatchGiftEntryHeader.test.js

```diff
@@ -51,9 +30,18 @@ describe('c-ge-batch-gift-entry-header', () => {
 
     describe('getRecord @wire data', () => {
         it('renders user record details', async () => {
-            getGiftBatchTotalsBy.mockResolvedValue(APEX_GIFT_BATCH_TOTALS_BY_SUCCESS);
             const element = setup();
 
+            element.batchTotals = {
+                hasValuesGreaterThanZero: true,
+                hasPaymentsWithExpiredAuthorizations: false,
+                totalGiftsCount: 20,
+                processedGiftsCount: 10,
+                failedPaymentsCount: 0,
+                failedGiftsCount: 5,
+                expiredPaymentsCount: 0
+            }
```

Comment:
This block isn't strictly necessary for this test since we're confirming that the expected title is rendered per the record that's returned by the getRecord wire adapter. `batchTotals` is optional for this test and I would recommend we remove it.

```suggestion
```